odl/test/discr/discr_utils_test.py

```diff
@@ -0,0 +1,265 @@
+# Copyright 2014-2020 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Unit tests for `discr_utils`."""
```

Comment:
Basically the same tests as `discr_mappings_test.py` before, with simplifications.